cmd/root/auth.go

```diff
@@ -83,12 +83,16 @@ func MustAccountClient(cmd *cobra.Command, args []string) error {
 		// 2. 99% of admins will have access to just one account
 		// hence, we don't need to create a special "DEFAULT_ACCOUNT" profile yet
 		_, profiles, err := databrickscfg.LoadProfiles(cmd.Context(), databrickscfg.MatchAccountProfiles)
-		if err != nil {
-			return err
-		}
-		if len(profiles) == 1 {
+		if err == nil && len(profiles) == 1 {
 			cfg.Profile = profiles[0].Name
 		}
+
+		if err != nil {
+			// if there is no config file, we don't want to fail and instead just skip it
+			if !errors.Is(err, databrickscfg.ErrNoConfiguration) {
```

Comment:
This can be collapsed in the outer if + should happen before doing anything with the output, IMO.